docker-compose.yaml

```
@@ -0,0 +1,11 @@
+version: '3.3'
+services:
+    symfony-docs:
+        build:
+            context: ./
+        image: symfony-docs:latest
+        user: ${CURRENT_UID}
+        ports:
+            - 8080:8000
```

Consider using `.env` file to inject environment variable here:
```suggestion
            - "${DOCS_PORT}:8000"
```
Otherwise, people that customize this file (when 8080 is occupied) may try to commit their changes unintentionally.

---

I believe that the best approach is to use `docker-compose.override.yml` as said by @Guikingone 
I will update the docs where it says to change the versioned `docker-compose.yml`, add `docker-compose.override.yml` to `.gitignore` and point people to update the override file instead.